README.Rmd

```diff
@@ -54,7 +54,7 @@ the reviewer to:
 
 Validation can serve as an umbrella for various terms, and admittedly, companies will diverge on what may be the "correct approach". The `riskassessment` app is built on a `rismetric`-foundation, whose developers follow the validation philosophy proposed in [this white paper](https://www.pharmar.org/white-paper/) published by the R Validation Hub. As such, the scope of `riskassessment` and `riskmetric` are only designed to support decision making from that view point. The full robustness and reliability of any software may (and likely will) require deeper inspection by the reviewing party.
 
-_Note:_ Development of both `riskassessment` and `riskmetric` were made possible thanks to the [R Validation Hub](https://www.pharmar.org/about/), a collaboration to support the adoption of R within a biopharmaceutical regulatory setting.
+_Note:_ Development of both `riskassessment` and `riskmetric` were made possible thanks to the [R Validation Hub](https://www.pharmar.org), a collaboration to support the adoption of R within a biopharmaceutical regulatory setting.
```

Comment:
@aclark02-arcus the about page is no longer there. I elected to redirect to the main page but wasn't sure if a different page was desired.